chainweb.openapi.yaml

```diff
@@ -816,7 +807,7 @@ components:
                 the respective node is using a valid CA signed SSL certificate.
 
               type: string
-              oneOf:
+              anyOf:
```

Comment:
`oneOf` means exactly one, not one or more

Reply:
You're right. I was wondering, then: how did this come up as a validation error? Turns out, the `format` specifiers are mostly not actually in the openapi spec, just things that various tools may or may not support, and in particular the library I'm using doesn't even try to check them. So they do all appear to overlap, because they're all trivial. For now, I suggest we stick with `anyOf` and add a TODO